sdk/servicebus/Azure.Messaging.ServiceBus/README.md

```diff
@@ -238,6 +238,23 @@ messages.Add(new ServiceBusMessage("Second"));
 // send the messages
 await sender.SendMessagesAsync(messages);
 ```
+### Receive a batch of messages
+
+In order to receive a batch of messages from queue, topic or subscription:
+
+```C# Snippet:ReceiveBatchOfMessages
+// create a receiver that we can use to receive the messages
```

Comment:
I believe that we'll want to include context here for creating the client and where `queueName` comes from.    Our goal is to allow each snippet to be as self-contained as possible so that it can be copy/pasted without the need to search through the sample for the related parts.

Reply:
Makes sense.  Reverted back to how it was with the old heading, just included the 3 missing lines that were supposed to show the batch receiving of messages.